lightning/src/ln/channelmanager.rs

```diff
@@ -550,6 +561,35 @@ impl_writeable_tlv_based_enum!(EventCompletionAction,
 	};
 );
 
+#[derive(Clone, PartialEq, Eq, Debug)]
+/// If something is blocked on the completion of an RAA-generated [`ChannelMonitorUpdate`] we track
+/// the blocked action here. See enum variants for more info.
+pub(crate) enum RAAMonitorUpdateBlockingAction {
```

Comment:
```suggestion
pub(crate) enum RAAMonitorUpdateBlockedAction {
```
? I think this commit might be a lot more comprehensible if we saved it for #2169